packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -108,15 +108,14 @@ export class OfficeAddinUsageData {
   public async reportEventApplicationInsights(eventName: string, data: object): Promise<void> {
     if (this.getUsageDataLevel() === UsageDataLevel.on) {
       const usageDataEvent = new appInsights.Contracts.EventData();
-      usageDataEvent.name = eventName;
+      usageDataEvent.name = `${eventName}${this.options.isForTesting ? "-test" : ""}`;
```

Comment:
This is minor but it would be cleaner to write the code this way:
```
usageDataEvent.name = this.options.isForTesting ? "${eventName}-test" : eventName;
```

Reply:
OK I will make that change but every incremental change of course requires verification